packages/app/src/components/TVChart/TVChart.tsx

```diff
@@ -191,7 +197,8 @@ export function TVChart({
 				{ text: '5D', resolution: '15', description: '5 Days' },
 				{ text: '30D', resolution: '1H', description: '30 Days' },
 				{ text: '3M', resolution: '1H', description: '3 Months' },
-			],
+			] as TimeFrameItem[],
+			...(chartData ? { saved_data: JSON.parse(chartData) } : {}),
```

Comment:
I think it'd be cleaner if we moved the json parsing to the loadChartData function. We could always return an object there and then on need for conditional here.

Reply:
Good point. Pushed.